db/migrate/20201201161307_remove_contents_and_lookup_from_downloads.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveContentsAndLookupFromDownloads < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :downloads, :lookup
```

Comment:
when this gets deployed, is this going to disrupt users who are in the middle of an export or haven't downloaded their finished export yet?

Reply:
Yeah I need to think about this. We can check if someone is doing an export easily, but i think the larger issue is getting access to previous exports that haven't been downloaded yet.

Reply:
would it be possible/make sense to remove support for the old download functionality in a later release?

Reply:
That's not a bad idea Tim, let's consider that. I think it could be accomplished with a conditional in the downloads controller. Otherwise we'll just need to let people know there will be a transition, and old downloads will expire with the new release.

Reply:
@holmesie and I had the discussion that we would just let users know that old downloads would be inaccessible after the release and any previously initiated exports that were not downloaded will need to be re-done.